questions/storeinventory.md

```diff
@@ -65,34 +65,63 @@ Please note that while an add-on isn't necessarily required for this solution, u
 
 Luckily in Java we don't have to go through what is essentially a custom hopper minecart, killing the player to get their items. Instead we can use `NBT` to solve this issue for us. _Usage of functions is highly recommended and is expected for this explanation._
 
-There are community made datapacks that do the heavy lifting for you like [PlayerDB](https://rx-modules.github.io/PlayerDB/).
+There are community made datapacks that do the heavy lifting for you like [PlayerDB](https://rx-modules.github.io/PlayerDB).
 
 ### Storing
```

Comment:
I think since storages are more clean (and performant) than marker entities, we should put the storage solution first, then the marker solution. Same below for the returning part.